lib/itax_code/encoder.rb

```diff
@@ -22,7 +22,7 @@ class Encoder
     # @option data [String]       :gender     The user gender
     # @option data [String, Date] :birthdate  The user birthdate
     # @option data [String]       :birthplace The user birthplace
-    def initialize(data = {}, utils = Utils.new)
+    def initialize(data = {}, utils = ItaxCode::Utils)
```

Comment:
The prefix is unnecessary here, as you're already inside the `ItaxCode` module.

```suggestion
    def initialize(data = {}, utils = Utils)
```